iOS-ssumgo/iOS-ssumgo/Source/Presentation/Notification/NotificationView.swift

```diff
@@ -0,0 +1,99 @@
+//
+//  NorificationView.swift
+//  iOS-ssumgo
+//
+//  Created by 서준영 on 11/17/24.
+//
+
+import SwiftUI
+
+struct NotificationView: View {
+    
+    private let rectWidth: CGFloat = 339
+    
+    var body: some View {
+        VStack(spacing: 0) {
+            BackNavigationBar(
+                rightIcon: false,
+                title: "알림"
+            )
+            Divider()
+            
+            ScrollView {
+                VStack(spacing: 0) {
+                    HStack {
+                        Text("읽지 않음")
+                            .font(.pretendard(.bold, size: 16))
+                            .foregroundStyle(.black)
+                            .frame(maxWidth: .infinity, alignment: .leading)
+                            .padding(.leading, 28)
+                        Spacer()
+                        CustomTextButton(
+                            title: "모두읽음",
+                            action: {
+                                print("모두읽음 버튼 클릭")
+                            },
+                            color: Color("s_main"),
```

Comment:
하드코딩(=문자열 기반 값 참조 방식)을 쓰게 되면 오타가 있을 경우, 런타임이 발생할 수도 있어서
```suggestion
                            color: .sMain,,
```
이런식으로 UIColor를 변환하여 사용하는 방식을 주로 사용합니다~!